src/lib/gundb/FeedStorage.js

```diff
@@ -405,18 +278,11 @@ export class FeedStorage {
     // not actually listening to this event
     if (eventsName.includes('PaymentWithdraw')) {
       const event = events.find(e => {
```

Comment:
as you're using loads you could simply do `const event = find(events, { name: 'PaymentWithdraw' })`